rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
Ah got it, when we did packer support a lot of the packer stuff wasn't there so this makes sense.  As long as users not leverage windows image support are not required to add in object id, no objections to PR.